assets/app/view/game/token_selector.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'view/game/hex'
+require 'lib/radial_selector'
+
+module View
+  module Game
+    class TokenSelector < Snabberb::Component
+      include Lib::RadialSelector
+      include Actionable
+      needs :tile_selector, store: true
+      needs :tokens
+      needs :city
+      needs :slot_index
+      TOKEN_SIZE = 40
+      SIZE = TOKEN_SIZE / 2
+      DISTANCE = TOKEN_SIZE
+
+      def render
+        tokens = list_coordinates(@tokens.keys, DISTANCE, SIZE).map do |corporation, left, bottom|
+          layable = @tokens[corporation].first
+          style = {
+            position: 'absolute',
```

Comment:
looks like this css is shared either create a class with the shared CSS in main.css or add a helper function to radial selector

Reply:
it's dynamic, so helper function